docs/rules/handle-done-callback.md

```diff
@@ -0,0 +1,68 @@
+# Enforces handling of callbacks for async tests (no-exclusive-tests)
+
+Mocha allows you to write asynchronous tests by adding a `done` callback to the parameters of your test function.
+It is easy to forget calling this callback after the asynchronous operation is done.
+
+Example:
+
+```js
+
+it('should work', function (done) {
+    fetchData(options, function (error, data) {
+        expect(error).not.to.be.ok;
+        expect(data).to.deep.equal({ foo: 'bar' });
```

Comment:
You could add a comment below this line like `// The done callback was not called` to make this even more clear.